docker/build-debian.sh

```diff
@@ -196,7 +196,17 @@ if [[ "${STAGE}" == 'deps' ]]; then
     echo
     echo -e "\e[1;32m Builder: installing gmock \e[0m"
     apt-get -y install libgtest-dev google-mock
-
+    # install gtest lib
+    cd /usr/src/googletest/googletest
+    cmake CMakeLists.txt
+    make
+    cp *.a /usr/lib
+    # install gmock lib
+    cd /usr/src/googletest/googlemock
+    cmake CMakeLists.txt
+    make
+    cp *.a /usr/lib
+    
```

Comment:
Interesting code and resolution of the travis problem. Nevertheless, take a look on the previous apt-get line...

Reply:
I`ve seen the apt-get line, but apparently it does not install the binaries(at least nowhere where I could have found them), therefore I create them here.